ethereum/PFP.sol

```diff
@@ -64,16 +64,16 @@ library MerkleProof {
 contract PFP is ERC721A, Ownable, VRFConsumerBase {
     using Strings for uint256;
     
-    uint256 public MAX_TOKENS = 20000;
-    string public baseExtension = ".json";
+    uint256 public constant MAX_TOKENS = 20000;
+    string public constant BASE_EXTENSION = ".json";
     string private _baseURIextended;
     address public _passAddress;
     bool public revealed = false;
     string public notRevealedUri = "";
     uint256 public randStartPos;
-    bytes32 public vrfKeyHash;
+    bytes32 public immutable vrfKeyHash;
```

Comment:
Can set this to immutable as it is set in the constructor and does not change (save 2,100 gas on read).